front/app/containers/EventsShowPage/components/EventSharingButtons/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+// components
+import SharingButtons from 'components/Sharing/SharingButtons';
+
+// i18n
+import { useIntl } from 'utils/cl-intl';
+
+// hooks
+import useLocalize from 'hooks/useLocalize';
+import useAuthUser from 'api/me/useAuthUser';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { IEventData } from 'api/events/types';
+import messages from './messages';
+
+interface Props {
+  event: IEventData;
+}
+
+const EventSharingButtons = ({ event }: Props) => {
+  const { formatMessage } = useIntl();
+
+  const { data: authUser } = useAuthUser();
+  const localize = useLocalize();
+
+  if (!isNilOrError(event)) {
+    const eventUrl = `${location.origin}/events/${event.id}`;
+    const eventTitle = localize(event.attributes.title_multiloc);
+    const eventLocation = localize(event.attributes.location_multiloc);
+    const utmParams = !isNilOrError(authUser)
```

Comment:
We don't need to use `!isNilOrError` anymore, just `authUser ? ... : ...` is enough